docs/rules/no-ignored-test-files.md

```diff
@@ -0,0 +1,92 @@
+# Ensure no tests are written in ignored files
+
+When searching for tests, AVA ignores files contained in folders named `fixtures` or `helpers`. By default, it will search in `test.js test-*.js test/**/*.js`, which you can override by specifying a path when launching AVA or in the [AVA configuration in the `package.json` file] ](https://github.com/sindresorhus/ava#configuration).
```

Comment:
True. I've added the mention about node_modules, and to clarify, a message sayig that they won't be linted due to ESLint.